### PR TITLE
[chore] adding readme on development tips

### PR DESCRIPTION
**Pull request by KurtStam**
fixes: https://github.com/syndesisio/syndesis/issues/88

### CODE REVIEW

File: app/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Development Tips
+
+## Start the Rest API w/o security
```
Comment (gashcrumb) [inline app/README.md]: Why is this needed?  Like what reason would I need to go through these steps?  How is this different than the normal development workflow?

File: app/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    the `apiEndpoint` needs to point to tinyproxy (`http://localhost:9090`):
+    
+        "apiBase": "http://localhost:9090"
+     
+     or
+
+        "apiEndpoint": "http://localhost:9090/api/v1",
+        "mapperEndpoint": "http://localhost:9090/mapper/v1",
+
+  5. Start the UI with from `$SYNDESIS/app/ui` (You may need to `npm rebuild node-sass`)
```
Comment (gashcrumb) [inline app/README.md]: If this is strictly related to the UI should this information maybe be added to `app/ui/README.md` instead?

File: app/README.md (hunk 1 above)
Comment (KurtStam) [reply app/README.md]: @gashcrumb It allows for quick turn around times when working on an issue that spans REST and UI. That said we should add the instructions for the `normal` workflow too if you can.

File: app/README.md (hunk 2 above)
Comment (KurtStam) [reply app/README.md]: @gashcrumb seems easier to me to keep it all in one place.

File: app/README.md (hunk 1 above)
Comment (gashcrumb) [reply app/README.md]: Yep, it's here -> https://github.com/syndesisio/syndesis/blob/master/app/ui/README.md#day-to-day-workflow

File: app/README.md (hunk 2 above)
Comment (gashcrumb) [reply app/README.md]: Shouldn't that be the 'developer handbook' then?  Not entirely sure where that gets it's content from...

**Comment by gashcrumb** [comment]
Aha, according to [this PR](https://github.com/syndesisio/syndesis/pull/679) we might want to add this info here somewhere -> https://github.com/syndesisio/syndesis/tree/master/doc/sdh

I'll create an issue to do the same with the readme under UI, that file needs to get cleaned up anyways as there's loads of stale information in there...

**Comment by gashcrumb** [comment]
Let's just merge this for now and I'll add a todo item on my issue regarding integrating this information...
